store/sqlstore/team_store.go

```diff
@@ -951,6 +951,7 @@ func (s SqlTeamStore) UpdateMember(member *model.TeamMember) (*model.TeamMember,
 	return members[0], nil
 }
 
+// GetMember returns a member based on the member's UserId
```

Comment:
I think you are assuming because you are in the TeamStore the `teamId` is obvious enough, but I think is better to be explicit, @jgilliam17 what do you think?

Reply:
:point_up: @justinegeffen

Reply:
Thanks, @jespino. Apologies for the delay, I missed this ping.

```suggestion
//GetMember returns a member based on the their teamId and userId
```
Not sure if "based on" is correct or we should say "using" but I see "based on" is used for GetTeamsForUser as well so 0/5 on that.